Fastfile: fix check for screenshots

AlvaroBrey opened the pull request:
We now have the app icon in the fastlane folder, so this check fails.


<!--
TESTING

Writing tests is very important. Please try to write some tests for your PR. 
If you need help, please do not hesitate to ask in this PR for help.

Unit tests: https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#unit-tests
Instrumented tests: https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#instrumented-tests
UI tests: https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#ui-tests
 -->
- [x] Tests written, or not not needed

tobiasKaminsky commented:
This check is/was to prevent me from accidentally uploading old screenshots.
Maybe we want use:
find metadata/android/*/images -type f | grep -vc icon.png

So that only "icon.png" is allowed.

AlvaroBrey commented:
> This check is/was to prevent me from accidentally uploading old screenshots. Maybe we want use: find metadata/android/*/images -type f | grep -vc icon.png
> 
> So that only "icon.png" is allowed.

Added back and fixed

AlvaroBrey commented:
/backport to stable-3.24

tobiasKaminsky commented on fastlane/Fastfile:
```diff
@@ -258,8 +258,8 @@ private_lane :checkLibrary_Final do
 desc "check if screenshots exists and exit"
-private_lane :checkIfScreenshotsExist do
-    sh(" if [ -e  metadata/android/*/images ] ; then echo 'Screenshots in fastlane folder exist; aborting!' ; exit 1 ; fi")
+private lane :checkIfScreenshotsExist do
```
is this private lane vs private_lane working?

AlvaroBrey replied:
Oops, no. That was a typo as I made it public to test it, and then messed up when restoring it. Fixed.
